exercise/api/csv/views.py

```diff
@@ -101,7 +101,7 @@ def list(
         queryset = Submission.objects.filter(
             exercise_id__in=ids,
             submitters__in=profiles
-        ).prefetch_related(Prefetch('exercise', BaseExercise.objects.all()), 'notifications', 'files')
+        ).prefetch_related('exercise', 'notifications', 'files')
```

Comment:
It is not obvious to me why the Prefetch is changed, but great that the shorter form works.

Reply:
There are a couple of similar changes in this PR. `Submission.exercise` is a ForeignKey field. Before commit 2, it would always use the base manager, even in prefetches. `BaseExercise.objects.all()` was used to specify that we want to use the default manager here. In commit 2, the field has been changed to use DefaultForeignKey, which uses the default manager while prefetching, so the longer form is no longer needed.